precise/util.py

```diff
@@ -79,7 +79,7 @@ def play_audio(filename: str):
 
 
 def activate_notify():
-    audio = 'data/activate.wav'
+    audio = 'precise/data/activate.wav'
```

Comment:
Any reason this is needed? In relation to `precise/utils.py` the file should still be `../data/activate.wav'.

Reply:
Yeah this is likely wrong. The issue is when using a pip install it fails to find the wav, right now I can't find the wav in the package at all so it was probably a fluke that this worked from the directory I ran things...

Gonna investigate again.

Reply:
I updated the location of the data folder to work better when installing through pip